src/App.tsx

```diff
@@ -80,14 +80,18 @@ const App: React.FC = () => {
             localStorage.setItem(GAME_URL, uri + "/");
           }
         });
-      CapApp.addListener("appStateChange", Util.onAppStateChange);
+      const urlParams = new URLSearchParams(window.location.search);
+      if (urlParams.get("continue")) {
```

Comment:
not every time we have this param "continue" after paying the game 
example if you go to leaderboard page there will be no "continue" param
make sure to have this "continue" param on all pages and once the page is refreshed remove this param